### DISCUSSION TITLE
"x commits to [branch] since this release" is missing, part 2

### DISCUSSION THREAD

**Discussion by dgw** [community/community#23890] [New to GitHub]
<p>I decided not to bump <a href="https://github.community/t/x-commits-to-master-since-this-release-is-missing/618">https://github.community/t5/How-to-use-Git-and-GitHub/x-commits-to-master-since-this-release-is-missing/m-p/3705</a> because it’s so old, but the issue is still present.</p>
<p>Either this feature was removed <em>for new releases</em> some time in the last 2 weeks or so (I have a release from 20 days ago that still shows this), or it’s related to something else.</p>
<p>Does GitHub only show “x commits to [branch] since this release” when the release was created from a branch, not a specific commit? In my release workflows, it’s very convenient to click the “x commits” link to see a comparison between that tag and the branch head, because that commit list is immediately usable to write the changelog entry for the next version. I’ll have to stop pointing releases to specific commits if this is the case.</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hi <a class="mention" href="/dgw">@dgw</a>, </p>
<p>The <code>x commits to master since this release</code> message should still exist, and will only appear if the target branch that’s selected when the release was created is <code>master</code> and the current state of the <code>master</code> branch in the repository has more commits than the release in question.</p>
<p>If those variables have been met but the release is still missing that text, could you please send over a link to the repository in question to <a href="https://github.com/contact" rel="nofollow noopener">https://github.com/contact</a> so that I can take a closer look?</p>

**Comment by dgw** [comment, marked as answer]
<p>Nah, it sounds like I’m experiencing a side effect of creating a release pointed at a specific commit instead of a branch. I’ll just have to remember not to do that if I want the tracking text. :slight_smile:</p>

**Comment by tapsboy** [comment]
<p>I actually see this gone in most of my repos</p>
<p>Used to be very handy. Is it really gone this time?</p>

**Comment by jkmathew** [comment]
<p><a class="mention" href="/AndreaGriffiths11">@AndreaGriffiths11</a><br>
This is still not visible. Is something changed recently?</p>

**Comment by gpuma** [comment]
<p>I also cannot see this anymore. What happened?</p>

**Comment by jendiamond** [comment]
<p><a href="https://github.com/UCLALibrary/ursus/releases" class="inline-onebox" rel="noopener nofollow ugc">Releases · UCLALibrary/ursus · GitHub</a> It is not visible in my repo either. It is more than a handy tool. Now it is a pain to see if there were commits since the last release. I really hope this feature is returned. My ream counted on it. We do releases all the time and used this almost daily.</p>

**Comment by david-poindexter** [comment]
<p>+1</p>
<p>I keep waiting for this feature to show back up, but it doesn’t.  It has been a real pain for releases since this feature has disappeared.  This was a nice way to quickly determine all that was done since the last release.</p>

**Comment by dereuromark** [comment]
They now completely removed it :cry: That's terrible, this was one of the most useful infos here in the releases page for maintainers!! Even the default branch already helped here..

**Reply by j0k3r** [reply]
I can't believe the link is gone. We now have to go to `/compare` to manually change the base to the latest tag ...

**Reply by jdr145** [reply]
It looks like it's back now!!

**Reply by dereuromark** [reply]
Hell yes! I guess crying loudly helped :P

**Comment by cartergray234** [comment]
I noticed this too. It seems GitHub may only show that link for releases created from a branch, not a specific commit. [Hopefully](https://dentalimplantcostcalculators.com/) GitHub can confirm if it's intentional or a bug.
